plugin/executor/containerd/containerd.go

```diff
@@ -52,10 +66,23 @@ func (e *Executor) Create(id string, spec specs.Spec, stdout, stderr io.WriteClo
 	ctx := context.Background()
 	err := e.client.Create(ctx, id, &spec, &opts)
 	if err != nil {
-		return err
+		status, _ := e.client.Status(ctx, id)
+		if status != libcontainerd.StatusRunning && status != libcontainerd.StatusUnknown {
+			if err2 := e.client.Delete(ctx, id); err2 != nil && !errdefs.IsNotFound(err2) {
+				logrus.WithError(err2).WithField("plugin", id).Error("Error cleaning up containerd container")
+			}
+			err = e.client.Create(ctx, id, &spec, &opts)
```

Comment:
I dont understand this additional call to `Create`.

Reply:
It's a recovery from left-over containerd containers. The bug leaves stuff behind on error, this cleans them up and tries again.

Reply:
Which bug?

Reply:
The bug being fixed by this PR, introduced by the containerd 1.0 integration.

Reply:
haha! I expected a bug number :)